packages/11ty/_includes/components/object-cards-list.webc

```diff
@@ -1,22 +1,46 @@
-<script webc:type="js">
-const getObjectCardElements = () => {
-  return objects.map((object) => {
-    const { id, imagePath, objectData, objectName, objectNameHtml, url } = object;
-    return `
-     <li data-object-id="${id}">
-       <object-card
-         image-path="${imagePath}"
-         object-name="${objectName}"
-         object-name-html="${objectNameHtml}"
-         url="${url}"
-         ${this.webc.renderAttributes(objectData)}
-       ></object-card>
-     </li>`
-  }).join('')
+<ol class="object-cards">
+  <li webc:for="object of objects" :data-object-id="object.id" @html="getObjectCard(object)"></li>
```

Comment:
Avoiding using a template and JavaScript rendered markup in that template may make these components easier to read, something like the following is likely easier to understand:
```suggestion
  <li webc:for="object of objects" :data-object-id="object.id">
    <object-card :url="object.url" $renderAttributes(object.objectData)></object-card>
  </li>
```